torchci/clickhouse_queries/weekly_force_merge_stats/query.sql

```diff
@@ -25,45 +25,60 @@ WITH issue_comments AS (
     FROM
         issue_comment FINAL
     WHERE
-        (
+        issue_comment.created_at >= {startTime: DateTime64(3)}
+        AND issue_comment.created_at < {stopTime: DateTime64(3)}
```

Comment:
Did you test if just moving these by themselves made a noticeable difference? I'm surprised if CH doesn't optimize this part already

Reply:
I did not and I don't think it makes an impact, just left it in there cause it makes it more readable imo

Reply:
Gotcha, I had thought this is what you were referring to with optimization #1 "Reordered time filters for earlier filtering"